app/controllers/quiz_game_search_controller.rb

```diff
@@ -0,0 +1,40 @@
+class QuizGameSearchController < ApplicationController
+
+  def create
+    if empty_quiz 
+      flash[:error] = "Please Tell Me a Fear"
+      redirect_to "/quiz"
+    else
+      games = create_game_objects(params)
+      create_redirect(games)
+    end
+  end
+
+  private 
+
+  def empty_quiz
+      boxes = params.keys.find_all {|param| param.include?("box")}
```

Comment:
boxes refer to the `check boxes id` from the fear quiz. just trying to pull out all the parameters that have the word `box`